scalalib/src/publish/VersionScheme.scala

```diff
@@ -0,0 +1,11 @@
+package mill.scalalib.publish
+
+sealed abstract class VersionScheme(val value: String)
+
+object VersionScheme {
+  case object Always extends VersionScheme("always")
+  case object EarlySemVer extends VersionScheme("early-semver")
+  case object PVP extends VersionScheme("pvp")
+  case object SemVerSpec extends VersionScheme("semver-spec")
+  case object Strict extends VersionScheme("strict")
+}
```

Comment:
I think it would be nice to have some more info what scheme means what, esp. what "PVP" stands for. A concrete link would be helpful, too.

Reply:
Done ✅